drivers/spider.py

```diff
@@ -0,0 +1,114 @@
+"""
+  Spider3 Rider Driver
+"""
+
+import sys
+import os
+import inspect
+
+OSGAR_ROOT = os.path.realpath(os.path.abspath(os.path.join(os.path.split(inspect.getfile( inspect.currentframe() ))[0],"..")))
+if OSGAR_ROOT not in sys.path:
+    sys.path.insert(0, OSGAR_ROOT) # access to logger without installation
+
+
+import serial
+import struct
+from threading import Thread, Event
+
+from lib.logger import LogWriter, LogReader
```

Comment:
I am starting to think that `from osgar.lib.logger ...` might not be that bad. If everything lived in top level `osgar` package, relative imports would work everywhere.

Reply:
I agree :-). I would do it as separate pull request (probably next week). New `osgar` subdirectory can then contain only new code and the remaining code in root will be deprecated and later removed.

Reply:
ok